src/Symfony/Component/Yaml/Tag/TagInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Yaml\Tag;
+
+/**
+ * @author Guilhem N. <egetick@gmail.com>
+ */
+interface TagInterface
+{
+    /**
+     * @param mixed $value
+     *
+     * @return mixed
+     */
+    public function construct($value);
```

Comment:
I think we should pass $tag and $value, so that one could create a single object to manage several tags if required.
$tag should match the key used when registering if possible (no namespace added/removed).
About "construct' => "createValue"?